cylc/rose/stem.py

```diff
@@ -17,24 +17,20 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
-"""rose stem [path]
+"""rose stem [path] [options]
 
-Install a rose-stem suite using cylc install.
+Rose stem:
+- Allows users to specify source trees to install using --source.
+- Provides Cylc install with standard jinja2 variables
+  These variables allow suite designers to group tasks for users to
+  specify with --group/--task.
 
-To run a rose-stem suite use "cylc play".
 
-Install a suitable suite with a specified set of source tree(s).
+Note: At Cylc 8, rose stem acts as a wrapper to Cylc install. You will
+need to use cylc play to start the workflow.
 
-Default values of some of these settings are suite-dependent, specified
-in the `rose-suite.conf` file.
 
-Examples
-
-    rose stem --group=developer
-    rose stem --source=/path/to/source --source=/other/source --group=mygroup
-    rose stem --source=foo=/path/to/source --source=bar=fcm:bar_tr@head
-
-Jinja2 Variables
+JINJA2 VARIABLES
```

Comment:
We don't use all-caps headings in other commands.

```suggestion
Jinja2 Variables
```

Reply:
Can these be moved until after the options? (as in Rose 2019)

Reply:
Not easily. The output is formatted by our option parser which prefers options at the end. I think there's an epilogue hook we could possibly use.